### PR TITLE
added new page called blog

### PR DESCRIPTION
**Pull request by Hodal66**
*USER*
should see now an articles
can comment on any article
can share the articles
**Manager**
Can view all the user suggestions like views, Comments, 
can also change the articles, update, delete articles

### CODE REVIEW

**Review by Salim-54 — COMMENTED** [review]
Great job!

File: css/blog.css [hunk 1]
```diff
@@ -0,0 +1,226 @@
...
+.title .sub_title a:hover{
+    color:#fff;
+}
+
+
+
+
+
+
+
```
Comment (Xldivin) [inline css/blog.css:171]: try to remove this whole space

Comment (Hodal66) [reply css/blog.css:171]: yes let me try but how can I make it look good???

**Comment by Hodal66** [comment]
yes
Thanks

**Pull request merged**